modules/apps/address/address-impl/src/main/java/com/liferay/address/internal/configuration/AdditionalCountriesConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.address.internal.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Loc Pham
+ */
+@ExtendedObjectClassDefinition(
+	category = "additional-address",
+	scope = ExtendedObjectClassDefinition.Scope.COMPANY
+)
+@Meta.OCD(
+	id = "com.liferay.address.internal.configuration.AdditionalCountriesConfiguration",
+	localization = "content/Language",
+	name = "additional-countries-configuration-name"
+)
+public interface AdditionalCountriesConfiguration {
+
+	@Meta.AD(
+		description = "enable-iso-countries-description",
```

Comment:
For me it's better to specify the iso code (so enable-iso-3166-countries-description / enable-iso-3166-countries / enableISO3166Countries().
What do you think?

In this way if in the future we will have also enable-customer-countries we can manage it in the same configuration with a different toggle but without confusing with names